intro-to-tests/practice-intro-to-pytest.md

```diff
@@ -864,6 +899,25 @@ class TestPython1(unittest.TestCase):
 ```
 
 ##### !end-tests
+##### !hint
```

Comment:
Arguably, the tests here require no more complex a solution than the previous question. If we are going to plant the TDD seed here that we really want to write the minimum code to make a test pass, but also want to encourage students to provide a real implementation here, then we should probably have both the even and odd test cases in the presented tests.